trySwift/SpeakerDetailViewController.swift

```diff
@@ -11,7 +11,7 @@ import TrySwiftData
 
 class SpeakerDetailViewController: UITableViewController {
 
-    var speaker: Speaker!
```

Comment:
Ah, ok. It is not used in another place, but if you open the `Speakers` tab on eg. iPad or iPhone Landscape, the `SplitViewController` will be able to show two `UIViewControllers`: in this case the `SpeakersViewController`, which is the table of speakers (the "master" view controller in `UISplitViewController` terms) and the `SpeakerDetailViewController`, which shows the speaker's details (the "detail" view controller in `UISplitViewController` terms).
See the image below.

The thing is... these master and detail view controllers get loaded at the same time. The detail view controller does its thing, but has not (and cannot) have received a `Speaker` from the master view controller yet, so it crashes.
![simulator screen shot 31 mar 2017 09 15 03](https://cloud.githubusercontent.com/assets/4190298/24540235/8bf88000-15f2-11e7-8411-84feaad2f89d.png)